### PR TITLE
Updates for docker

### PR DESCRIPTION
**Pull request by nberard**
for prod: 

```
docker run -ti -e "RABBITMQ_HOST=vippriv-brokk-mq.mutu.prod.canaltp.fr" -v /tmp/stat_logger:/var/logs/navitia-stat-logger navitia-stat-logger:wip
```

### CODE REVIEW

File: stat_logger/daemon.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 import os
 import platform
 import gzip
```
Comment (vincentlepot) [inline stat_logger/daemon.py]: gzip is then useless

File: stat_logger.conf.yml [hunk 2]
```diff
@@ -2,10 +2,10 @@ storage:
   hdfs: false
   localfs: true
 localfs:
-  root_dir: "/tmp"
+  root_dir: <%= ENV['LOGS_DIRECTORY'] %>
```
Comment (vincentlepot) [inline stat_logger.conf.yml]: Maybe we can just have this one as a constant and declare it as a volume in the image. We then mount a volume on this given path. No need for a configuration here.

Comment (nberard) [inline stat_logger.conf.yml]: ok for var/logs/navitia-stat-logger ?

Comment (vincentlepot) [inline stat_logger.conf.yml]: Sounds good

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    ./configure && \
+    make && \
+    make install && \
+    ldconfig && \
+    rm -rf /var/lib/protobuf-2.6.1 /tmp/protobuf-2.6.1.tar.gz
+
+ADD requirements.txt /tmp/requirements.txt
+RUN pip install -qr /tmp/requirements.txt && rm -f /tmp/requirements.txt
+
+ADD . /opt/navitia-stat-logger
```
Comment (vincentlepot) [inline Dockerfile]: You need to also compile the protobuf files.

**Pull request merged**